tutorial/markdown/java/spring-boot.md

```diff
@@ -53,172 +91,494 @@ mvn package
+```
+
 ## What We'll Cover
 
-A simple REST API using Spring Boot and the Couchbase SDK version 3.x with the following endpoints:
+A simple REST API using Spring Boot and the `Couchbase SDK version 3.x` with the following endpoints:
 
-- [POST a Profile](#post-a-profile) – Create a new user profile
-- [GET a Profile by Key](#get-a-profile-by-key) – Get a specific profile
-- [PUT Profile](#put-profile) – Update a profile
-- [DELETE Profile](#delete-profile) – Delete a profile
-- [GET Profiles by Searching](#get-profiles-by-searching)  – Get all profiles matching first or last Name
+- [Create new airlines with essential information](#post-mapping).
+- [Update airline details](#put-mapping).
+- [Delete airlines](#delete-mapping).
+- [Retrieve airlines by ID](#get-mapping).
+- [List all airlines with pagination](#get-mapping).
+- [List airlines by country](#custom-sql-queries).
+- [List airlines by destination airport](#custom-sql-queries).
 
-## Document Structure
+### Document Structure
 
-We will be setting up a REST API to manage some profile documents. Our profile document will have an auto-generated UUID for its key, first and last name of the user, an email, and hashed password. For this demo we will store all profile information in just one document in a collection named `profile`:
+We will be setting up a REST API to manage some airline documents. Our airline document will have a structure similar to the following:
 
 ```json
 {
-  "pid": "b181551f-071a-4539-96a5-8a3fe8717faf",
-  "firstName": "John",
-  "lastName": "Doe",
-  "email": "john.doe@couchbase.com",
-  "password": "$2a$10$tZ23pbQ1sCX4BknkDIN6NekNo1p/Xo.Vfsttm.USwWYbLAAspeWsC"
+  "id": "airline_8091",
+  "type": "airline",
+  "name": "Couchbase Airways",
+  "callsign": "Couchbase",
+  "iata": "CB",
+  "icao": "CBA",
+  "country": "United States",
+  "active": true
 }
 ```
 
-As we can see, we want our user's password to be encrypted in the database too, we can achieve this simply with bcrypt, a dependency we have installed.
+The `id` field is the unique identifier for the document. The `type` field is used to identify the type of document. The `name` field is the name of the airline. The `callsign` field is the callsign of the airline. The `iata` field is the IATA code of the airline. The `icao` field is the ICAO code of the airline. The `country` field is the country of the airline. The `active` field is a boolean value indicating whether the airline is active or not.
+
+### Code Organization
+
+- `src/test/java`: Contains integration tests.
+- `src/main/java/org/couchbase/quickstart/springboot/repositories`: Contains the repository implementation.
+- `src/main/java/org/couchbase/quickstart/springboot/models`: Contains the data model.
+- `src/main/java/org/couchbase/quickstart/springboot/controllers`: Contains the RESTful API controllers.
+- `src/main/java/org/couchbase/quickstart/springboot/services`: Contains the service classes.
+
+## Data Model
```

Comment:
Check the location of this section with other tutorials

Reply:
put in just below the lets review the code just like the python quickstart